src/_components/guides/category.rb

```diff
@@ -0,0 +1,5 @@
+class Guides::Category < Bridgetown::Component
+  def initialize(resources:, title:, css_classes: "", position: nil)
+    @resources, @title, @css_classes, @position = resources, title, css_classes, position
```

Comment:
According to our guides, we should avoid multiple assignments per line:https://github.com/bump-sh/guides/tree/master/ruby#ruby
```suggestion
    @resources = resources
    @title = title, 
    @css_classes = css_classes
    @position = position
```